lib/ClangImporter/ImportDecl.cpp

```diff
@@ -8464,11 +8464,12 @@ ClangImporter::Implementation::createConstant(Identifier name, DeclContext *dc,
     // Create the expression node.
     StringRef printedValueCopy(context.AllocateCopy(printedValue));
     if (value.getKind() == clang::APValue::Int) {
-      if (type->getCanonicalType()->isBool()) {
+      if (type->getCanonicalType()->isBool() ||
+          value.getInt().getBitWidth() == 1) {
```

Comment:
Is the first check for `isBool()` necessary or is the second check for `getBitWidth() == 1` sufficient?

Reply:
Because this is a more general API (not just used for enums) I wasn't sure if all users would pass it an APInt where `bitwidth = 1`. I'll try removing the check and see if the tests still all pass.

Reply:
The tests passed so I removed the first condition.